string new_greeting
---
string old_greeting